projects/core/components/dialog/dialog.service.ts

```diff
@@ -1,24 +1,24 @@
-import {Injectable} from '@angular/core';
-import {AbstractTuiDialogService} from '@taiga-ui/cdk';
+import {Inject, inject, Injectable} from '@angular/core';
+import {AbstractTuiDialogService, TuiIdService} from '@taiga-ui/cdk';
 import {TuiDialogOptions} from '@taiga-ui/core/interfaces';
+import {TUI_DIALOG_OPTIONS} from '@taiga-ui/core/tokens/dialog-options';
 import {PolymorpheusComponent} from '@tinkoff/ng-polymorpheus';
 
 import {TuiDialogComponent} from './dialog.component';
 
 const DIALOG = new PolymorpheusComponent(TuiDialogComponent);
-const DEFAULT_OPTIONS = {
-    size: `m`,
-    required: false,
-    closeable: true,
-    dismissible: true,
-    label: ``,
-    header: ``,
-} as const;
 
 @Injectable({
     providedIn: `root`,
 })
 export class TuiDialogService extends AbstractTuiDialogService<TuiDialogOptions<any>> {
```

Comment:
it turns out this token will work only for this class?

<img width="669" alt="image" src="https://user-images.githubusercontent.com/12021443/187300097-c8461046-6da7-447d-8e5e-dc982478e4c7.png">

Reply:
I mean we have to do the same everywhere?

<img width="649" alt="image" src="https://user-images.githubusercontent.com/12021443/187300448-b0cf9062-12a2-49e0-b619-7ea033484e8e.png">

Reply:
what do you think if we can just override some properties if we need it
<img width="963" alt="image" src="https://user-images.githubusercontent.com/12021443/187301366-9332000e-da4e-4a80-8bd7-f68cab5d1354.png">

Reply:
without additional changes in inherited classes

Reply:
While it is a very good idea for reducing code duplication, I don't think having only one token is feasible. There would be the issue of conflicting property names between different dialog options?

Reply:
> While it is a very good idea for reducing code duplication, I don't think having only one token is feasible. There would be the issue of conflicting property names between different dialog options?

That's right. This token is for exactly this dialog implementation.